src/uct/rocm/ipc/rocm_ipc_cache.c

```diff
@@ -217,7 +217,7 @@ ucs_status_t uct_rocm_ipc_create_cache(uct_rocm_ipc_cache_t **cache,
         goto err_destroy_rwlock;
     }
 
-    cache_desc->name = strdup(name);
+    cache_desc->name = ucs_strdup(name, "rocm cache->name");
```

Comment:
no need strdup - can pass name directly

Reply:
"rocm_ipc_cache_name"

Reply:
> no need strdup - can pass name directly

will not work:
https://github.com/openucx/ucx/blob/209aae62d5aff3b4b5da07697c5240cdb9985990/src/uct/rocm/ipc/rocm_ipc_ep.c#L21
name is on-stack variable

Reply:
> "rocm_ipc_cache_name"

renamed